stacker/providers/aws.py

```diff
@@ -5,10 +5,23 @@
 
 from . import exceptions
 from .base import BaseProvider
+from ..util import retry_with_backoff
 
 logger = logging.getLogger(__name__)
 
 
+def retry_cf_throttling(fn, attempts=3, args=None, kwargs=None):
```

Comment:
small nit: i like the `retry_with_backoff` function name because it describes what it does. this function, a little less so, `retry_on_throttling` maybe?

not a big deal